### PR TITLE
latest/fixed version installers (windows)

### PR DESCRIPTION
**Pull request by maximlopin**

### CODE REVIEW

File: distributions/windows/install.py [hunk 1]
```diff
@@ -11,11 +11,17 @@
 GET_PIP_URL = 'https://bootstrap.pypa.io/get-pip.py'
 VC_REDIST_URL = 'https://aka.ms/vs/16/release/vc_redist.x64.exe'
 
+# must be replaced in another script
+NAME = '$name'
+VERSION = '$version'
+
+assert NAME != '$' + 'name'
+assert VERSION != '$' + 'version'
```
Comment (George3d6) [inline distributions/windows/install.py:19]: Why are these here ?

File: distributions/windows/install.py [hunk 2]
```diff
@@ -11,11 +11,17 @@
...
+
 if len(sys.argv) < 2:
-    INSTALL_DIR = 'C:/Program Files/MindsDB'
+    INSTALL_DIR = os.path.join('C:\PROGRA~1', NAME)
```
Comment (George3d6) [inline distributions/windows/install.py:22]: Is `PROGRA~1` the correct location ? looks like a weirdly named dir

**Review by George3d6 — CHANGES_REQUESTED** [review]

File: distributions/windows/install.py (hunk 2 above)
Comment (maximlopin) [reply distributions/windows/install.py:22]: it resolves to Program Files, I'm using it because otherwise there is a weird error because of spaces in the dir name

File: distributions/windows/install.py (hunk 1 above)
Comment (George3d6) [reply distributions/windows/install.py:19]: Ok, so it's basically a feature for when testing a new version, got it.
